setup.py

```diff
@@ -473,6 +477,40 @@ def run(self):
 
             self.copy_file(export_lib, target_lib)
 
+    def build_extension(self, ext):
+        if ext.name == 'torch._C':
+            cmake_build_dir = os.path.join(self.build_lib, 'torch')
```

Comment:
TODO: build somewhere else and copy selected files into this directory?

Reply:
Best would be to build in the `build/` directory at the root level, since build_deps seems to use that as well.